test/services/MeasureService.test.ts

```diff
@@ -0,0 +1,44 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../../src/config/serverConfig';
+import { cleanUpDb, testSetup } from '../utils';
+import supertest from 'supertest';
+
+let server: Server;
+
+const Measure: fhir4.Measure = { resourceType: 'Measure', id: 'measure123', status: 'active' };
+
+describe('MeasureService', () => {
+  beforeAll(async () => {
+    const config = serverConfig;
+    server = initialize(config);
```

Comment:
Also, is this really necessary?

```suggestion
    server = initialize(serverConfig);
```